packages/back-end/src/api/openapi/schemas/ExperimentWithDecisionCriteriaStatus.yaml

```diff
@@ -0,0 +1,18 @@
+allOf:
+  - $ref: "./Experiment.yaml"
+  - type: object
+    properties:
+      decisionCriteriaStatus:
```

Comment:
hmm I think this name is not entirely correct -- we might have a status that is not powered by the decision criteria at all.

For the front-end we use:
```
export type ExperimentResultStatus =
  | DecisionFrameworkExperimentRecommendationStatus
  | { status: "no-data" }
  | { status: "unhealthy"; unhealthyData: ExperimentUnhealthyData }
  | { status: "before-min-duration" };
```

Which is the underlying data that we use to figure out the label (`Running`, `Stopped`, etc) + detailed status and tooltip if needed.

Maybe should use `detailedStatus` ? (which might be powered by decision criteria but might not?)